src/bundles/Elsa.WorkflowServer.Web/appsettings.json

```diff
@@ -15,6 +15,10 @@
   "ConnectionStrings": {
     "Sqlite": "Data Source=elsa.sqlite.db;Cache=Shared;"
   },
+  "MongoDb": {
+    "ConnectionString": "mongodb://localhost:27017",
+    "DatabaseName": "elsa-workflows"
+  },
```

Comment:
Would it be nice if we could support using the "ConnectionStrings" section for consistency with other persistence stores? E.g.

```json
"ConnectionStrings": {
    "Sqlite": "Data Source=elsa.sqlite.db;Cache=Shared;"
   "MongoDb": "mongodb://localhost:27017/elsa-workflows"
  }
```